package.json

```diff
@@ -2,25 +2,30 @@
   "name": "agoda-tslint",
   "version": "0.3.1",
   "description": "A set of TSLint rules used on some Agoda projects.",
-  "repository": "https://github.com/agoda-com/agoda-tslint.git", 
+  "repository": "https://github.com/agoda-com/agoda-tslint.git",
   "author": "Maciej Malkowski <maciej.malkowski@agoda.com>",
   "main": "agoda-tslint.json",
   "license": "Apache-2.0",
   "private": false,
   "scripts": {
-    "build": "yarn tslint && tsc",
-    "tslint": "node node_modules/tslint/lib/tslint-cli.js -p tsconfig.json"
+    "build": "yarn lint && tsc",
+    "lint": "tslint -p tsconfig.json",
```

Comment:
So, I am not sure about the current CI on this repo. We will have to check that nothing is currently running `yarn tslint`